src/doc/trpl/arrays-vectors-and-slices.md

```diff
@@ -49,7 +49,7 @@ languages.
 
 A *vector* is a dynamic or "growable" array, implemented as the standard
 library type [`Vec<T>`](../std/vec/) (we'll talk about what the `<T>` means
-later). Vectors always allocate their data on the heap. Vectors are to slices 
+later). Vectors always allocate their data on the heap. Vectors are to arrays
```

Comment:
Actually, I believe it is correct. I believe it is trying to say "A vector `Vec<T>` is to a slice `&[T]` what `String` is to `&str`". Probably it would be helpful to use the syntax for the types to clarify.

In particular, I think we are trying to call a fixed-length array type like `[T;N]` an "array" and one where the length is dynamically known `[T]` a "slice".